projects/kit/components/line-clamp/line-clamp.component.ts

```diff
@@ -124,22 +117,26 @@ export class TuiLineClamp implements DoCheck, AfterViewInit {
         const {scrollHeight, scrollWidth} = this.outlet.nativeElement;
         const {clientWidth} = this.el;
 
-        return scrollHeight > this.maxHeight() || scrollWidth > clientWidth;
+        return (
+            scrollHeight > parseFloat(this.style.maxHeight) || scrollWidth > clientWidth
```

Comment:
nit: maybe save maxHeight to a const to avoid wrapping return to multiple lines for readibility?

Reply:
fixed